github/messages.go

```diff
@@ -117,3 +147,44 @@ func validateSignature(signature string, payload, secretKey []byte) error {
 	}
 	return nil
 }
+
+// getWebhookEvent parses the payload and request, and build the Event.
+func getWebhookEvent(messageType string, payload []byte) (interface{}, error) {
```

Comment:
Rename to `parseWebhookEvent`, that seems like a much more fitting and consistent prefix than "get".

Also, should it be "Webhook" or "WebHook"? I see both used in this PR. But the rest of the codebase is also inconsistent. The exported code seems to use "WebHook", so we should probably go with that (since it's harder to change compared to documentation).